scipy/spatial/kdtree.py

```diff
@@ -222,6 +222,9 @@ class KDTree(cKDTree):
         is the boxsize along i-th dimension. The input data shall be wrapped
         into :math:`[0, L_i)`. A ValueError is raised if any of the data is
         outside of this bound.
+    build_workers : int, optional
```

Comment:
Just to be clear. On my side I am just commenting on the interface to make it coherent with such existing options.

The doc should be adapted as well.
```suggestion
    workers : int, optional
```